frontend/main.js

```diff
@@ -240,13 +242,17 @@ async function startApp () {
       }
       sbp('okTurtles.events/off', CONTRACT_IS_SYNCING, initialSyncFn)
       sbp('okTurtles.events/on', CONTRACT_IS_SYNCING, syncFn.bind(this))
-      sbp('okTurtles.events/on', LOGIN, () => {
+      sbp('okTurtles.events/on', LOGIN, async () => {
         this.ephemeral.finishedLogin = 'yes'
 
         if (this.$store.state.currentGroupId) {
           this.initOrResetPeriodicNotifications()
           this.checkAndEmitOneTimeNotifications()
         }
+        const databaseKey = `chelonia/persistentActions/${sbp('state/vuex/getters').ourIdentityContractId}`
+        sbp('chelonia.persistentActions/configure', { databaseKey })
+        await sbp('chelonia.persistentActions/load')
+        await sbp('chelonia.persistentActions/retryAll')
```

Comment:
I think `'chelonia.persistentActions/load'` should automatically start the retries

Reply:
That would be fine, but on the other hand `/save` doesn't stop the pending retries

Reply:
It shouldn't. The persistent action queue is something that is always active as long as it has actions inside of it while the app is running.